src/vendor/studio.scss

```diff
@@ -1845,3 +1886,42 @@ button {
     }
   }
 }
+
+.dropdown-group-wrapper {
+  position: relative;
+
+  .pgn__form-control-decorator-trailing {
+    @include font-size(22);
+
+    .btn-icon.btn-icon-sm {
+      width: 3.6rem;
+      height: 3.6rem;
+    }
+
+    .pgn__icon.btn-icon__icon {
+      height: 2.4rem;
+      width: 2.4rem;
+    }
+  }
+}
+
+.dropdown-container {
+  box-shadow: 0 2px 4px rgba(0, 0, 0, 0.15), 0 2px 8px rgba(0, 0, 0, 0.15);
+  border-radius: 4px;
+  max-height: 200px;
+  font-size: 1rem;
+  font-weight: normal;
+  line-height: 1.25rem;
+  overflow-y: scroll;
+  position: absolute;
+  background-color: #fff;
```

Comment:
Please replace `#fff` to `$white` or something else suitable

Reply:
replaced